README.rst

```diff
@@ -162,29 +162,8 @@ Transformer Engine comes preinstalled in the pyTorch container on
 From source
 ^^^^^^^^^^^
 
-For JAX and Tensorflow, pybind11 must be installed:
+`See the installation guide <docs/installation>`__.
```

Comment:
Ok, on second thought I think pointing to a local file is more effort than it's worth. This link is technically correct, but GitHub doesn't actually support displaying RST files and the link breaks when displayed with Markdown. Since this is the front page of our GitHub, I think having a working link is more important than having the most up-to-date install docs (which I don't expect to change much anyways):
```suggestion
`See the installation guide <https://docs.nvidia.com/deeplearning/transformer-engine/user-guide/installation.html>`__.
```

Reply:
It is like that in the commit. So resolving.